src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved */
-import * as React from 'react';
 import Tippy, { TippyProps } from '@tippyjs/react';
-import 'tippy.js/dist/tippy.css?external';
+import * as React from 'react';
```

Comment:
NIP: why do you move this down? 
Not a hard requirement, as we don't have linting rules for import order, but I think this import is always the first in this codebase.